### ISSUE TITLE
L2 M1 - Validación Cruzada

### ISSUE THREAD

**Issue by IvoVuskovic97** [IIC2115/Syllabus#86]
Hola tengo una duda al momento de realizar validación cruzada.

En el enunciado dice "Para definir los conjuntos de prueba y validación, independiente del tamaño que tengan, considere siempre el x% mas reciente de los registros, y no particiones aleatorias."

Pero al momento de realizar validación cruzada se debe dividir en k el set de entrenamiento, ocupando k-1 como entrenamiento y uno como validación en forma iterativa hasta que todos los sets tengan su turno como set de validación. En ese caso sería imposible cumplir la restricción del enunciado de solo poner los datos más recientes en validación

Algo similar ocurre en Leave One Out

Saludos

**Comment by pcseisdedos** [comment]
Claro, pero ese x% es para separar en prueba y validación, lo que tu dices ocurre con el set de validación.

Saludos,
Pablo 😁👍
